apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -211,35 +210,25 @@ export default function CurriculumQuickAssign({
       <CurriculumQuickAssignTopRow
         showPlOfferings={showPlOfferings}
         marketingAudience={marketingAudience}
-        updateMarketingAudience={updateMarketingAudience}
+        updateMarketingAudience={setMarketingAudience}
       />
-      {marketingAudience && !isPlOrHoc() && courseOfferings && (
-        <QuickAssignTable
+      {marketingAudience && courseOfferings && (
+        <SelectedQuickAssignTable
```

Comment:
this is nice!

Reply:
Thanks! Just trying to make things a bit DRYer.